stdlib/collections/__init__.pyi

```diff
@@ -37,7 +37,19 @@ else:
 
 class UserDict(MutableMapping[_KT, _VT]):
```

Comment:
Apparently this works, but it might be good to change this to be more explicit.

```suggestion
class UserDict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
```